tensorbay/client/segment.py

```diff
@@ -318,7 +318,7 @@ def upload_file(self, local_path: str, target_remote_path: str = "") -> None:
 
         checksum = self._calculate_file_sha1(local_path)
 
-        post_data["key"] = checksum
+        post_data["key"] = f'{permission["extra"]["objectPrefix"]}{checksum}'
```

Comment:
if there are only two strings need to be joined, `a + b` is faster than `f"{a}{b}"`